aws/resource_aws_elasticache_replication_group.go

```diff
@@ -232,6 +235,22 @@ func resourceAwsElasticacheReplicationGroup() *schema.Resource {
 	}
 }
 
+func resourceAwsElasticacheReplicationGroupMigrateState(v int, inst *terraform.InstanceState, meta interface {}) (*terraform.InstanceState, error) {
+	switch v {
+	case 0:
+	log.Printf("[WARN] Elasticache Replication Group (%s) v%d; migrating to v1 (noop)", inst, v)
+
+	fallthrough
```

Comment:
What is the reason for the `fallthrough`?  so still return 0?

Reply:
Yeah! It's so that as new versions are added it goes through them all the way up to the current version, which returns the instance itself (case `1` in this example)